crates/model/src/trade.rs

```diff
@@ -30,7 +33,7 @@ pub struct Trade {
     // Settlement Data
     pub tx_hash: Option<H256>,
     // Fee Policy Data
-    pub fee_policies: Vec<FeePolicy>,
+    pub fee_policies: Vec<(FeePolicy, Option<ExecutedFee>)>,
```

Comment:
worth having its own struct since it is not just `fee_policies` anymore

Reply:
Can we make this not optional? Given that we are not really using the API yet, I think we can simply not expose any FeePolicy in case we don't have the executed amounts in the db yet (this way we don't force someone to handle the case where the policy is there but no amounts, which at least going forward is an invalid state).

Reply:
resolved